packages/datetime/test/common/dateUtilsTests.tsx

```diff
@@ -221,4 +221,119 @@ describe("dateUtils", () => {
             assertTimeIs(time, 11, 20);
         });
     });
+
+    describe("convert24HourMeridiem", () => {
+        it("returns given hour, if hour is AM and toAm", () => {
+            for (let hour = 0; hour <= 11; hour++) {
```

Comment:
for all the tests you've added here, please just test one or two explicit values. 
we gain nothing by iterating through math, but we lose much in readability.

try this:
```ts
expect(DateUtils.convert24HourMeridiem(4, true)).to.equal(4)
expect(DateUtils.convert24HourMeridiem(16, true)).to.equal(4) // < edge cases! tests!
```

Reply:
this comment still stands. please address it in all tests.